app/models/occupation.rb

```diff
@@ -0,0 +1,11 @@
+class Occupation < ApplicationRecord
+  validates :rapids_code, presence: true, uniqueness: true
+
+  def title_aliases=(list)
+    if list.is_a?(String)
+      super list.split(/;\s+/)
+    else
+      super list
```

Comment:
As an `array` column, does this automatically do a `<<` or does it just overwrite as it would any other column? I'm assuming anything we pass to `#title_aliases=` just overwrites what's in there previously. 

If so, we might eventually want to support a separate method for `#add_alias=` that'll simply add to the contest of the array. No need to do so now just thinking about how this might be used beyond the admin tools.

Reply:
Yes, you can use `<<` if you like on the array column.  Or you can set it directly with `title_aliases=`.